### PR TITLE
Adding GraphQL CMS and Theme Attributes to storeConfig

### LINKED ISSUE #3342: GraphQL: Add documentation for storeConfig queries in 2.3.1
# Community engineering issue

Type of Doc:
- [ ] New topic
- [x] Topic Update, Devdoc link:
- [ ] Other

Magento 2 GitHub Issue: 
https://github.com/magento/graphql-ce/issues/128

Magento 2 GitHub PR: https://github.com/magento/graphql-ce/pull/182

Magento 2 version:
- [ ] 2.1.X
- [ ] 2.2.X
- [x] 2.3.1
- [ ] back port
- [ ] forward port

Notifications of Changes:
- [ ] Changes to Admin interface?
- [ ] New software requirements?
- [x] New REST endpoints or updated GraphQL query syntax?
- [ ] New command line option?

## Content

GraphQL PR-182 adds the `StoreConfig` type to the `ThemeGraphQl` and `CmsGraphQl` modules. The documentation for the `storeConfig` queries will need to be updated, but I don't know its source module right now.

### PR DESCRIPTION
**Pull request by erikmarr**
## This PR is a:

- [ ] New topic
- [x] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will fix #3342

## Additional information

List all affected URLs 

- https://devdocs.magedevteam.com/2.3.1-integration/guides/v2.3/graphql/reference/store-config.html

### CODE REVIEW

File: guides/v2.3/graphql/reference/store-config.md [hunk 1]
```diff
@@ -1,38 +1,80 @@
...
+Use the `cms` attributes to retrieve information about the store's default pages.
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`cms_home_page` | String | Returns the name of the CMS page that identifies the home page for the store
+`cms_no_cookies` | String | Identifies a specific CMS page that appears when cookies are not enabled for the browser
+`cms_no_route` | String | Identifies a specific CMS page that you want to appear when a 404 “Page Not Found” error occurs
+`front` | String | Indicates the landing page that is associated with the base URL
+`no_route` | String | Contains the URL of the default page that you want to appear when if a 404 “Page not Found” error occurs
+`show_cms_breadcrumbs` | Int | Determines if a breadcrumb trail appears on all CMS pages in the catalog. Options: Yes / No
```
Comment (keharper) [inline guides/v2.3/graphql/reference/store-config.md]: The value can't be Yes/No when the value is an Int

File: guides/v2.3/graphql/reference/store-config.md [hunk 2]
```diff
@@ -1,38 +1,80 @@
 ---
 group: graphql
-title: storeConfig endpoint
+title: Store endpoint
 ---
 
-The `storeConfig` endpoint returns information about a store's configuration, including its locale, currency codes, and the secure and unsecure URLs.
+The `Store` endpoint defines information about a store's configuration.
```
Comment (keharper) [inline guides/v2.3/graphql/reference/store-config.md]: Add a link to https://devdocs.magento.com/guides/v2.3/graphql/send-request.html with an explanation that the Store header allows you to query a non-default store.

File: guides/v2.3/graphql/reference/store-config.md [hunk 3]
```diff
@@ -1,38 +1,80 @@
...
+`secure_base_static_url` | String | The secure fully-qualified URL that specifies the location of static view files, such as `https://magentohost.example.com/pub/static/`
+`secure_base_url` | String | The store's fully-qualified secure base URL, such as `https://magentohost.example.com/`
+`timezone` | String | The store's time zone, such as `America/Chicago`
+`website_id` | Integer | The ID number assigned to the parent website  
+`weight_unit` | String | The weight unit for products, such as `lbs` or `kgs`
+{:style="table-layout:auto;"}
+
+### Supported theme attributes
+
+Use the `theme` attributes to retrieve information about the store's thematic elements; such as, footer and header information, copyright text, and logo information.
```
Comment (keharper) [inline guides/v2.3/graphql/reference/store-config.md]: Mention these attributes are defined in the ThemeGraphQl module.

File: guides/v2.3/graphql/reference/store-config.md [hunk 4]
```diff
@@ -1,38 +1,80 @@
...
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`absolute_footer` | String | Contains scripts that must be included in the HTML before the closing &lt;body&gt; tag
+`copyright` | String | The copyright statement that appears at the bottom of each page
+`default_description` | String | The description that provides a summary of your site for search engine listings and should not be more than 160 characters in length
+`default_keywords` | String | A series of keywords that describe your store, each separated by a comma
+`default_title` | String | The title that appears at the title bar of each page when viewed in a browser
+`demonotice` | Int | Controls the display of the demo store notice at the top of the page. Options include: Yes / No
```
Comment (keharper) [inline guides/v2.3/graphql/reference/store-config.md]: The value can't be Yes/No when the value is an Int

File: guides/v2.3/graphql/reference/store-config.md [hunk 5]
```diff
@@ -1,38 +1,80 @@
...
+`weight_unit` | String | The weight unit for products, such as `lbs` or `kgs`
+{:style="table-layout:auto;"}
+
+### Supported theme attributes
+
+Use the `theme` attributes to retrieve information about the store's thematic elements; such as, footer and header information, copyright text, and logo information.
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`absolute_footer` | String | Contains scripts that must be included in the HTML before the closing &lt;body&gt; tag
```
Comment (keharper) [inline guides/v2.3/graphql/reference/store-config.md]:
```suggestion
`absolute_footer` | String | Contains scripts that must be included in the HTML before the closing `<body>` tag
```

Comment (keharper) [reply guides/v2.3/graphql/reference/store-config.md]: You don't need to escape the less/greater than signs.

File: guides/v2.3/graphql/reference/store-config.md [hunk 6]
```diff
@@ -1,38 +1,80 @@
...
+`copyright` | String | The copyright statement that appears at the bottom of each page
+`default_description` | String | The description that provides a summary of your site for search engine listings and should not be more than 160 characters in length
+`default_keywords` | String | A series of keywords that describe your store, each separated by a comma
+`default_title` | String | The title that appears at the title bar of each page when viewed in a browser
+`demonotice` | Int | Controls the display of the demo store notice at the top of the page. Options include: Yes / No
+`head_includes` | String | Contains scripts that must be included in the HTML before the closing &lt;head&gt; tag
```
Comment (keharper) [inline guides/v2.3/graphql/reference/store-config.md]:
```suggestion
`head_includes` | String | Contains scripts that must be included in the HTML before the closing `<head>` tag
```

File: guides/v2.3/graphql/reference/store-config.md [hunk 7]
```diff
@@ -1,38 +1,80 @@
...
+`logo_height` | Int | The height of your logo image in pixels
+`logo_width` | Int | The width of your logo image in pixels
+`title_prefix` | String | A prefix that appears before the title to create a two- or three-part title
+`title_suffix` | String | A suffix that appears after the title to create a two-or three part title
+`welcome` | String | Text that appears in the header of the page and includes the name of customers who are logged in
 {:style="table-layout:auto;"}
 
+### Supported CMS attributes
+
+Use the `cms` attributes to retrieve information about the store's default pages.
```
Comment (keharper) [inline guides/v2.3/graphql/reference/store-config.md]: Mention these attributes are defined in the `CmsGraphQl` module

File: guides/v2.3/graphql/reference/store-config.md (hunk 5 above)
Comment (erikmarr) [reply guides/v2.3/graphql/reference/store-config.md]: fixed

File: guides/v2.3/graphql/reference/store-config.md (hunk 6 above)
Comment (erikmarr) [reply guides/v2.3/graphql/reference/store-config.md]: fixed

File: guides/v2.3/graphql/reference/store-config.md (hunk 3 above)
Comment (erikmarr) [reply guides/v2.3/graphql/reference/store-config.md]: fixed

File: guides/v2.3/graphql/reference/store-config.md (hunk 7 above)
Comment (erikmarr) [reply guides/v2.3/graphql/reference/store-config.md]: fixed

File: guides/v2.3/graphql/reference/store-config.md (hunk 1 above)
Comment (erikmarr) [reply guides/v2.3/graphql/reference/store-config.md]: fixed

File: guides/v2.3/graphql/reference/store-config.md (hunk 4 above)
Comment (erikmarr) [reply guides/v2.3/graphql/reference/store-config.md]: fixed

File: guides/v2.3/graphql/reference/store-config.md (hunk 2 above)
Comment (erikmarr) [reply guides/v2.3/graphql/reference/store-config.md]: fixed

**Comment by erikmarr** [comment]
running tests

**Pull request merged**